app/assets/javascripts/models/data/dataset.js.coffee

```diff
@@ -97,7 +97,13 @@ ns.Dataset = do (ko
 
     thumbnail: ->
       granule = @browseable_granule
-      "#{scalerUrl}/#{granule}?h=85&w=85" if granule?
+      collection_id = @browseable_collection
```

Comment:
The way we have things now, collections show the browse image from their first granule. This logic won't work for that.